src/web/web_templates.ml

```diff
@@ -3762,7 +3762,7 @@ let booth () =
       div ~a:[a_style "text-align:center; margin-bottom:20px;"] [
         span ~a:[a_id "progress1"; a_style "font-weight:bold;"] [txt (s_ "Input credential")];
         txt " — ";
-        span ~a:[a_id "progress2"] [txt (s_ "Answer to questions")];
+        span ~a:[a_id "progress2"] [txt (s_ "Answers to questions")];
```

Comment:
Or?

Reply:
I would keep "Answer to questions" here, as all other items in the breadcrumb are verbs in infinitive form. Or all items should be switched to nouns.

Reply:
For another day.